rollup.config.js

```diff
@@ -33,7 +33,7 @@ export default {
     plugins: [
         resolve(),
         postcss({
-            extract: 'styles.css',
+            extract: 'dist/chrome/styles.css',
```

Comment:
I don't think this is supposed to change, it's probably that you're running V2 of the plugin (instead of v3) you might need to run `npm install` to get the latest version

Reply:
Ah, knew I'd missed something! Right will update 👍🏼

Reply:
hmm somehow you ended up with a styles.css in the root directory, you should probably remove `node_modules` and reinstall